lib/src/base-styles/_nx-colors.scss

```diff
@@ -554,6 +570,22 @@
     --nx-color-status-indicator-error-icon-background: var(--nx-swatch-red-40);
     --nx-color-status-indicator-error-icon-border: var(--nx-color-status-indicator-error-icon-background);
 
+    // NxGlobalSidebar
+    --nx-color-global-sidebar-background: var(--nx-color-component-background);
+    --nx-color-global-sidebar-separator-color: var(--nx-swatch-indigo-25);
+    --nx-color-global-sidebar-menu-icon-border-focus: var(--nx-color-interactive-border-focus);
+    --nx-color-global-sidebar-menu-icon-color: var(--nx-swatch-indigo-10);
+    --nx-color-global-sidebar-menu-icon-background: var(--nx-color-interactive-border-focus);
```

Comment:
This is actually only used on hover and active and should maybe be named accordingly

Reply:
> This is actually only used on hover and should maybe be named accordingly

Corrected name here: b208f29c7d45c5c6a9431853a768a737eb66bc02